devApps/XFormsApp/SecondPage.cs

```diff
@@ -40,7 +40,7 @@ public class SecondPage : ContentPage
 
         public string User = "<User>";
         private string Tenant = "<Tenant>";
-        public const string AndroidBrokerRedirectURI = "msauth://com.microsoft.xformsdroid.adal/mJaAVvdXtcXy369xPWv2C7mV674=";
+        public const string AndroidBrokerRedirectURI = "adaliosapp://com.yourcompany.xformsapp";
```

Comment:
>adaliosapp [](start = 56, length = 10)

nit: seems off to call the protocol ios app for android

Reply:
it is, this change was made for testing. it has been reverted.